library/std/src/fs.rs

```diff
@@ -358,32 +358,26 @@ impl File {
         OpenOptions::new().write(true).create(true).truncate(true).open(path.as_ref())
     }
 
-    /// Returns a new OpenOptions object.
+    /// Returns a builder struct to configure how a file is opened.
```

Comment:
It is not a builder sturct.

Reply:
`OpenOptions` struct is a builder struct. I'm not sure we want to repeatedly mention
`OpenOptions`. I'll leave this for reviewer to decide.